packages/main/src/BusyIndicator.js

```diff
@@ -97,6 +97,23 @@ class BusyIndicator extends UI5Element {
 		this.i18nBundle = getI18nBundle("@ui5/webcomponents");
 	}
 
+	onEnterDOM() {
```

Comment:
We can also set tabindex="-1" for this case on the host and it should remove the content of the busy from the tab chain.

Reply:
We added it because it covers another case, but it is not enough.